messaging/src/main/java/org/axonframework/eventhandling/processors/streaming/pooled/SplitTask.java

```diff
@@ -120,14 +120,19 @@ private boolean splitAndRelease(Segment segmentToSplit) {
                 unitOfWorkFactory
```

Comment:
Again not really a `TokenStore` concern, more an `PooledStreamingEventProcessor` concern. I don't think we need to use the `UnitOfWorkFactory` here, right? Or am I missing something obvious, @MateuszNaKodach?